packages/template/-private/dsl/emit.d.ts

```diff
@@ -133,7 +137,25 @@ export declare function applySplattributes<
  *     <div foo={{bar}}></div>
  *     <AnotherComponent foo={{bar}} />
  */
-export declare function applyAttributes(element: Element, attrs: Record<string, AttrValue>): void;
+
+type WithSvgStrings<T> = {
+  [P in keyof T]?: T[P] extends SVGAnimatedString ? string : T[P];
+};
+
+// TODO: improve this to allow other keys
```

Comment:
anyone knowns how to achieve this?
basically would like to allow other keys that default to AttrValue.

Currently I do it with `Record<string, AttrValue>`, but that sets all properties to AttrValue